UPGRADING.INTERNALS

```diff
@@ -102,6 +102,10 @@ PHP 8.4 INTERNALS UPGRADE NOTES
    - The configure option --with-oci8 has been removed.
    - The configure option --with-zlib-dir has been removed.
    - The configure option --with-kerberos has been removed.
+   - The configure option --with-openssl-dir has been removed. SSL support in
+     ext/ftp and ext/mysqlnd is enabled implicitly, when building with
+     ext/openssl (--with-openssl), or explicitly by using new configure options
+     --with-ftp-ssl and --with-mysqlnd-ssl.
```

Comment:
The configure options are actually user facing changes that we need to document in manual

Reply:
Yes. PR to the doc-en coming up.